core/domain/role_services.py

```diff
@@ -108,6 +129,17 @@
 }
 
 
+def get_human_readable_role(role):
+    """Converts role to human readable format.
+
+    Args:
+        role: str. The role to convert.
+    Returns:
+        str. Role in human readable format.
+    """
+    return role.lower().replace('_', ' ')
```

Comment:
I thought we were going to have this use an explicit dict, rather than a calculation.